src/physics/site_types/spinhalf.jl

```diff
@@ -104,6 +94,12 @@ end
 op!(Op::ITensor,t::SiteType"S=1/2",
     ::OpName"S²",s::Index) = op!(Op,t,OpName("S2"),s)
 
+
+# Support the tag "SpinHalf" as equivalent to "S=1/2"
+
+space(::SiteType"SpinHalf"; kwargs...) = space(SiteType("S=1/2");kwargs...)
+state(::SiteType"SpinHalf",n::StateName) = state(SiteType("S=1/2"),n)
```

Comment:
This reminded me that we could also support `"S=½"`.

Reply:
I'd be ok with adding that. I'd have to learn how to type that though!

Reply:
```julia
help?> ½
"½" can be typed by \1/2<tab>
```